src/components/ec-table-sort/ec-table-sort.vue

```diff
@@ -61,8 +57,9 @@ const directionTitle = computed(() => {
   return 'Not sorted';
 });
 
-function onSort() {
-  emit('sort', props.direction);
+function onSort(): void {
+  // eslint-disable-next-line @typescript-eslint/no-non-null-assertion
+  emit('sort', props.direction!);
```

Comment:
```suggestion
  emit('sort', props.direction);
```
If you use withDefaults on line 38